ibis/sql/postgres/tests/test_functions.py

```diff
@@ -545,7 +546,12 @@ def test_category_label(alltypes, df):
     labels = ['a', 'b', 'c', 'd']
     bucket = d.bucket(bins)
     expr = bucket.label(labels)
-    result = expr.execute().astype('category', ordered=True)
+    result = expr.execute()
+
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore')
+        result = result.astype('category', ordered=True)
```

Comment:
Do we test against versions of pandas that not compatible with using the API suggested by the warning?

Reply:
I wasn't sure, that's why I'm filtering them. I can check though.

Reply:
Just tried, on python 2.7 (pandas 0.20.3) tests `pd.api.types.CategoricalDtype` doesn't have arguments.
I fallback to the old style and filter out the warnings instead.